### PR TITLE
Update Arabic lang.json

### PR DESCRIPTION
**Pull request by shawkynasr**
Arabic version
Long sentences need RTL ( Right to Left).

### CODE REVIEW

File: admin/src/lang.json [hunk 1]
```diff
@@ -692,32 +692,70 @@
...
-        "ctrlF": "Firefox افتح في",
-        "ctrlI": "Brave افتح في",
-        "ctrlT": "Tab Space افتح"
+        "ctrlC": "فتح في Chrome",
+        "ctrlW": "فتح في Edege",
+        "ctrlF": "فتح في Firefox",
+        "ctrlI": "فتح في Brave Browser",
+        "ctrlT": "فتح Tab Space",
+        "ctrlB": "Save selected text to Bear",
+        "goTip": "إذا كنت تريد فقط تعيين Tab Space كصفحة افتراضية لعلامة تبويب جديدة ، فيرجى استخدام العنوان \ "http://mytab.space/redirect.html\".",
```
Comment (yuanzhoucq) [inline admin/src/lang.json:749]: Thanks for your contribution!  Just the whitespace in `\ "` needs to be deleted. 😄

Comment (shawkynasr) [reply admin/src/lang.json:749]: Got it, thanks

**Comment by yuanzhoucq** [comment]
Thanks 😄

**Comment by shawkynasr** [comment]
Can you share screenshot of beta version for Arabic lang? I guess Arabic language need to span for RTL.

**Pull request merged**